draft-ounsworth-cfrg-kem-combiners.mkd

```diff
@@ -192,13 +191,12 @@ informative:
       -
         ins: E. Yogev
         name: Eylon Yogev
-        ins: 
 
 --- abstract
 
 The migration to post-quantum cryptography often calls for performing multiple key encapsulations in parallel and then combining their outputs to derive a single shared secret.
 
-This document defines a flexible multi-share KEM combiner to join an arbitrary number of key shares, that is a multi-PRF compatible with NIST SP 800-56Cr2 [SP800-56C]. The combiner defined here is CCA-secure as long as at least one of the ingredient
+This document defines a comprehensible and easy to implement Keccak-based KEM combiner to join an arbitrary number of key shares, that is compatible with NIST SP 800-56Cr2 [SP800-56C] when viewed as a key derivation function. The combiner defined here is a multi-PRF and CCA-secure as long as at least one of the ingredient
```

Comment:
Maybe we should better write "practical multi-PRF" as Andreas did initially (see sec. considerations), as Keccak is not a random function but indifferentiable up to some work load depending on the capacity.

Reply:
Done.